pyproject.toml

```diff
@@ -39,8 +39,8 @@ test = [
 ]
 
 [project.urls]
-changelog = "https://pyproject-metadata.readthedocs.io/en/stable/changelog.html"
-homepage = "https://github.com/FFY00/python-pyproject-metadata"
+changelog = "https://pep621.readthedocs.io/en/stable/changelog.html"
```

Comment:
I think this one is supposed to change to a pypa.io URL, but it hasn't yet. Having a working old URL is better than one that doesn't work.